front/src/modules/ui/utilities/scroll/hooks/useListenToScroll.ts

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+import debounce from 'lodash.debounce';
+
+export const StyledAnimatedScrollBarContainer = styled.div`
+  overflow: auto;
+
+  ::-webkit-scrollbar {
+    width: 0px;
```

Comment:
Adding `transition: width 5s ease-in-out;` here appeared to have no effects, also doing this with `framer-motion` is not straightforward.